### PR TITLE
Run Random Beacon Docs: section about staking in KEEP token dashboard

### PR DESCRIPTION
**Pull request by pdyraga**
Extracted and slightly reworked some parts from https://github.com/keep-network/keep-core/pull/1455 to cover KEEP token dashboard terms, staking, and authorizations.

### CODE REVIEW

File: docs/run-random-beacon.adoc [hunk 1]
```diff
@@ -6,6 +6,46 @@
 :numbered:
 toc::[]
 
+== Staking
+
+=== Terminology
```
Comment (Shadowfiend) [inline docs/run-random-beacon.adoc:11]: I would actually move this to the end <_<

Comment (pdyraga) [reply docs/run-random-beacon.adoc:11]: On it.

Comment (Shadowfiend) [reply docs/run-random-beacon.adoc:11]: Eh, I actually wonder if this whole section shouldn't be its own doc linked from here. This doc is about operating the client 🤔

Comment (pdyraga) [reply docs/run-random-beacon.adoc]: Yeah, I think we should give https://github.com/keep-network/keep-core/pull/1455 more attention though I think it makes sense to have it here temporarily before we can get staking doc out. What do you think?

Comment (Shadowfiend) [reply docs/run-random-beacon.adoc]: Yeah that's probably fair. At least it'll give us something to point at.

Comment (pdyraga) [reply docs/run-random-beacon.adoc]: Moved to the bottom.

**Pull request merged**